features/widows-orphans.yml

```diff
@@ -0,0 +1,7 @@
+name: widows and orphans
```

Comment:
Names are sentence-case, so should capitalize `widows`.

However, I realized that's not clearly articulated in the docs, just something I picked up in reviews. Filed #1804 for that.